### PR TITLE
CRM-21489 document new constant in civicrm.settings.php

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Document constant from https://github.com/civicrm/civicrm-packages/pull/197 in civicrm.settings.php

Comments
----------------------------------------
This is just signposting / documenting a change discussed elsewhere - ie. JIRA & the other PR

---

 * [CRM-21489: Deadlock retries have been accidentally blocked](https://issues.civicrm.org/jira/browse/CRM-21489)

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
test this please

File: templates/CRM/common/civicrm.settings.php.template [hunk 1]
```diff
@@ -420,6 +420,13 @@ if (!defined('CIVICRM_DB_CACHE_PREFIX')) {
 // define('CIVICRM_GETTEXT_NATIVE', 1);
 // }
 
+/**
+ * Define how many times to retry a transaction when the DB hits a deadlock
+ * (ie. the database is locked by another transaction). This is an
+ * advanced setting intended for high-traffic databases & experienced developers/ admins.
+ */
+define('CIVICRM_DEADLOCK_RETRIES', 3);
```
Comment (mattwire) [inline templates/CRM/common/civicrm.settings.php.template]: Is this adding the define in when it wasn't already added?  Or was it defined/defaulted elsewhere?

Comment (eileenmcnaughton) [reply templates/CRM/common/civicrm.settings.php.template]: it's new - but requires the merge in packages too

**Comment by seamuslee001** [comment]
Associated packages PR has been merged so merging this one

**Pull request merged**